src/main/resources/META-INF/rewrite/java-version-11.yml

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright 2022 the original author or authors.
+# Copyright 2024 the original author or authors.
```

Comment:
For any future PR: no need to update these copyright headers; we typically keep the year on when they were first introduced.